packages/playground/website/cypress/e2e/website-ui.cy.ts

```diff
@@ -22,9 +22,11 @@ describe('Playground website UI', () => {
 	// Test all PHP versions for completeness
 	describe('PHP version switcher', () => {
 		SupportedPHPVersions.forEach((version) => {
-			if (version === '7.0') {
-				// The SQLite integration plugin uses `private const` which is not supported in PHP 7.0
-				// @TODO: Adjust the plugin and remove this condition.
```

Comment:
This was fixed in https://github.com/WordPress/sqlite-database-integration/pull/82